AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2491,7 +2491,7 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 {
   __ASDisplayNodeCheckForLayoutMethodOverrides;
 
-  return CGSizeZero;
+  return constrainedSize;
```

Comment:
Should we check if this is a valid size before returning it? Seems like we should return CGSizeZero if it's invalid.